GVFS/GVFS.Installer.Mac/uninstall_vfsforgit.sh

```diff
@@ -54,18 +54,18 @@ function UnInstallVFSForGit()
         for uid in $(ps -Ac -o uid,command | grep -iw "loginwindow" | awk '{print $1}'); do
             unloadCmd="sudo launchctl bootout gui/$uid ${LAUNCHAGENTDIRECTORY}/$SERVICEAGENTLAUNCHDFILENAME"
             echo "$unloadCmd..."
-            eval $unloadCmd || exit 1
+            eval $unloadCmd || echo "Error: Could not unload ${LAUNCHAGENTDIRECTORY}/$SERVICEAGENTLAUNCHDFILENAME. Try logout and login to finish un-install."
```

Comment:
If we reach these echo statements, will we still return a non-zero error code?

Reply:
I would expect a non-zero error code if we didn't completely succeed, I think.

Reply:
how about we cache the error. But rather than exiting immediately, continue with remaining un-install. In the end exit with the cached error code.

Reply:
Updated to log & early exit in case of errors.